Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ however, it has to be formatted properly to pass verification tests.
 
 - Added `InputSystem.runUpdatesInEditMode` to enable processing of non-editor updates without entering playmode (only available for XR).
 - Added method `SetMotorSpeedsAndLightBarColor` as a workaround for setting both the light bar and motor speeds simultaneously on a DualShock 4 controller ([case 1271119](https://issuetracker.unity3d.com/issues/dualshock4-setlightbarcolor-and-setmotorspeeds-cannot-be-called-on-the-same-frame-using-input-system)).
+- Improved performance of `Touchscreen` by merging consecutive touch move events together.
```

Comment:
Too sparse. Think for this feature, we should add something to `Events.md` that explains what it does and how to turn it off.

Reply:
Agree, probably worth a separate manual page.
Maybe makes sense to add manual when gamepad stuff is also in?

Reply:
How great is our confidence it'll make it in time for 1.1-pre.6? If it doesn't make it, think it'd be good to have the docs go in first.

Reply:
I'll see what I can do

Reply:
Added manual